packages/runtime/src/useCases/transport/files/UploadOwnFile.ts

```diff
@@ -53,7 +52,7 @@ class Validator extends SchemaValidator<UploadOwnFileValidatableRequest> {
             );
         }
 
-        if (input.expiresAt && DateTime.fromISO(input.expiresAt) <= DateTime.utc()) {
+        if (input.expiresAt && CoreDate.from(input.expiresAt).isSameOrBefore(new CoreDate())) {
```

Comment:
please use CoreDate.now() instead of new CoreDate, IMO this is way more descriptive.

Reply:
I guess you mean CoreDate.utc(), don't you?

Reply:
yep